week4/MealsharingNET/Models/Meal.cs

```diff
@@ -0,0 +1,19 @@
+namespace MealsharingNET.Models;
+
+public class Meal
+{
+    public int ID { get; set; }
+    public string Title { get; set; }
+    public string Description { get; set; }
+    public string Location { get; set; }
+    public DateTime When { get; set; }
```

Comment:
Should this be in the Reservation instead of the Meal?

Reply:
When you create a meal event, you want to have a date when is going to be hosted, that is why the when is there.